src/modules/Packages.rb

```diff
@@ -18,6 +18,10 @@ class PackagesClass < Module
     RESOLVABLE_TYPES = [:product, :patch, :package, :pattern, :language]
     # Minimum set of packages required to enable VNC server
     VNC_BASE_PACKAGES = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    # Additional packages needed to run second stage in graphical mode
+    # Use exact package name (i.e. libyui-qt6 instead of libyui-qt) to prevent
+    # the UI from asking for confirmation
+    AUTOYAST_X11_PACKAGES = ["libyui-qt6", "yast2-x11"]
```

Comment:
I'd rather avoid `libyui-qt6` as it will be changed at some point, I'd rather use something more stable, like `yast2-qt` (see https://github.com/libyui/libyui-qt/blob/master/libyui-qt.spec.in#L51). @mvidner: or is there any better solution?